### ISSUE TITLE
Cannot tab to links in the "about" dialog

### ISSUE THREAD

**Issue by jobara** [phetsims/john-travoltage#211]
The links in the about dialog do not seem to be keyboard accessible. When the dialog initially opens the "Terms, Privacy & Licensing" link is focused, so you can open that with the keyboard; however, you cannot reach the others and if you move focus off the "Terms, Privacy & Licensing" link, you can't go back without closing and reopening the dialog.

I tested with Safari 10.0.3 on macOS 10.12.3

**Comment by terracoda** [comment]
@jobara, that's strange, I was able to Tab to all the thinks and read the content in the About dialog when using Voice Over on the same platform.

**Comment by jobara** [comment]
@terracoda i wasn't using VoiceOver for this issue. Could you try without VoiceOver with just keyboard navigation and see if you can reproduce it?

**Comment by jessegreenberg** [comment]
I am testing on Safari 9.0.1, but links are focusable for me, with and without VoiceOver.

@jobara this sounds exactly like what I was seeing in Firefox before changing user settings. Is something like this the cause?

http://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox

Are links focusable for you in other web pages?

**Comment by jobara** [comment]
@jessegreenberg it does work, I probably forgot to press option-tab when navigating. Although I find the tab order is a bit unexpected with the "close" button being the last item in the tab order. Anyways, I'll close this specific issues as it's not a bug.
